syft/format/internal/spdxutil/helpers/download_location.go

```diff
@@ -10,6 +10,7 @@ import (
 
 const NONE = "NONE"
 const NOASSERTION = "NOASSERTION"
+const SUPPLIERORG = "Organization"
```

Comment:
BTW We didn't have these constants when first implementing SPDX, but today there are constants in `tools-golang` for these.